torchao/quantization/subclass.py

```diff
@@ -191,19 +191,46 @@ def __torch_dispatch__(cls, func, types, args, kwargs):
             )
 
 
+@torch._dynamo.allow_in_graph
+def from_qtensor_components_int8dyn(
+    *args,
+    **kwargs
+):
+    return Int8DynamicallyQuantizedLinearWeight(
+        *args,
+        **kwargs,
+    )
+
+class ConstructTensorSubclassInt8Dyn(torch.nn.Module):
```

Comment:
Can this made generic for all tensor subclasses?

Reply:
can't do it now because of https://github.com/pytorch/pytorch/issues/124735, should be able to do it after this is fixed